frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard.tsx

```diff
@@ -84,41 +83,23 @@ export const CreateInternalCluster: React.FC<CreateInternalClusterProps> = ({
   const hasEnabledCreateStep =
     !!(state.nodes.length >= MINIMUM_NODES && scName && state.kms.hasHandled) &&
     hasConfiguredNetwork;
-  const { getStep, getParamString, getIndex, getAnchor } = navUtils;
-
-  const toLink = (steps: any, currentStep: string, modes: any) =>
-    getAnchor(getIndex(steps, currentStep, 2), getIndex(modes, modes.INTERNAL));
+  const { getStep, getParamString, getIndex } = navUtils;
 
   const steps: WizardStep[] = [
     {
-      name: (
-        <Link to={toLink(CreateStepsSC, CreateStepsSC.STORAGEANDNODES, MODES)}>
-          {' '}
-          {t('ceph-storage-plugin~Capacity and nodes')}{' '}
-        </Link>
-      ),
+      name: t('ceph-storage-plugin~Select capacity and nodes'),
       id: CreateStepsSC.STORAGEANDNODES,
       component: <SelectCapacityAndNodes state={state} dispatch={dispatch} mode={mode} />,
       enableNext: !!(state.nodes.length >= MINIMUM_NODES && scName),
     },
     {
-      name: (
-        <Link to={toLink(CreateStepsSC, CreateStepsSC.CONFIGURE, MODES)}>
-          {' '}
-          {t('ceph-storage-plugin~Security and network')}{' '}
-        </Link>
-      ),
+      name: t('ceph-storage-plugin~Configure'),
```

Comment:
Similar, it is `Security and network` not `Configure`... it was changed around 4.8 (https://issues.redhat.com/browse/RHSTOR-1670)